interface/src/app/gene/searchMany/gene-search-panel.tpl.html

```diff
@@ -14,4 +14,9 @@
       </search-result-table>
     </div>
   </div>
+  <div class="panel-footer">
+    <button type="button" class="btn btn-link" ng-click="switchToFew()">
+      Switch to searching with autocomplete.
+    </button>
```

Comment:
Do you thin kit makes more sense to put this button at the end of the whole page (instead of putting it in the middle of "Search for Genes" section and "Selected Genes" section)?